src/PhpSpec/Matcher/ThrowMatcher.php

```diff
@@ -121,11 +121,25 @@ public function verifyPositive($callable, array $arguments, $exception = null)
         }
 
         if (!$exceptionThrown instanceof $exception) {
+            $format = 'Expected exception of class %s, but got %s.';
+
+            // Show full error message if the thrown exception is an instance of Error.
```

Comment:
This is good stuff but you don't need the explanation here IMO, git history's enough.